hack/ci-build-image.sh

```diff
@@ -8,5 +8,6 @@ if [[ ! -z "$TRAVIS_PULL_REQUEST_BRANCH" ]]; then
   makeopts="-e GIT_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH"
 fi
 
+$make $makeopts bpftraceimage/build # HACK since the ubuntu image isn't pushed yet
```

Comment:
It seems this was necessary to get the CI to have the tag locally in its cache, since there is nothing for it to pull.

Reply:
Just pushed `quay.io/iovisor/kubectl-trace-bpftrace-base-ubuntu:49ff42b4e311794cddad24a848b3c225a1b61804"` you can remove this now